### PR TITLE
Some README improvements with grammar etc

### PR DESCRIPTION
**Pull request by intergrav**
I read through and made changes by myself, and then I used Grammarly to double-check. Let me know if you think there are any errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,31 +22,31 @@ Simple and easy to use Minecraft client for 1.8.9 (and in future™, newer versi
 ![Mods](./assets/screenshots/Mods.png)
 
 ## Pojav
-People have alreading been talking about this client working on Pojav, but it's not quite ready.
+People have already been talking about this client working on Pojav, but it's not quite ready.
 However, it does mostly work so you can download a preview from [here](https://github.com/Sol-Client/Installer/releases).
 
 ## Licence
```
Comment (osfanbuff63) [inline README.md:28]:
```suggestion
## License
```

**Comment by TheKodeToad** [comment]
Alreading :P.
I don't know how I missed it.

File: README.md [hunk 2]
```diff
@@ -55,14 +55,14 @@ Here are the main features currently available in the stable version:
 - Item physics.
 - Hypixel Additions.
 - OptiFine, downloaded automatically from the official site (open an issue if this is a bad thing).
-- Crosshair mod, allowing you to customise your crosshair, while fitting with the vanilla style.
-- Quick Play Mod. Allows you to quickly join games at the press of a button (by default, "M"). The key opens a menu where you can search for games, navigating through them using the arrow keys. If you type nothing, you can see recent games and a categorised list of all games.
+- Crosshair mod, allowing you to customize your crosshair while fitting with the vanilla style.
+- Quick Play Mod. Allows you to quickly join games at the press of a button (by default, "M"). The key opens a menu where you can search for games, navigating through them using the arrow keys. If you type nothing, you can see recent games and a categorized list of all games.
```
Comment (TheKodeToad) [inline README.md:59]: I'd prefer the UK spelling of these words.

File: README.md [hunk 3]
```diff
@@ -55,14 +55,14 @@ Here are the main features currently available in the stable version:
...
 - Customisable launcher servers - automatically detected from the game, with no pinned servers.
 - Better item tooltips - show item damage and more.
-- Symbol picker (happy, sad, and the extemely important snowman).
+- Symbol picker (happy, sad, and the extremely important snowman).
 - Chat channel display and switcher (you can still use the command).
-- Pop-up events (pop-up friend request, etc.).
-- Customisable font and colour scheme.
+- Pop-up events (pop-up friend requests, etc.).
+- Customisable font and color scheme.
```
Comment (TheKodeToad) [inline README.md:65]: Same here.

File: README.md [hunk 4]
```diff
@@ -78,7 +78,7 @@ If there are any issues with us using your code, please open an issue.
 ## Safety
 The code for this client is available, so if you have sufficient knowledge you can check yourself for nasty stuff.
 
-This client has been tested by multiple people on Hypixel. On most popular servers, disallowed mods are disabled. There are unknown servers, but just disable the correct mods and you should be fine. All mods are designed (and intend) not to change the behaviour of the player.
+This client has been tested by multiple people on Hypixel. On most popular servers, disallowed mods are disabled. There are unknown servers, but just disable the correct mods and you should be fine. All mods are designed (and intended) not to change the behavior of the player.
```
Comment (TheKodeToad) [inline README.md:81]: ... and here.
Also, I don't think intended needs to be in the past tense.

**Review by TheKodeToad — CHANGES_REQUESTED** [review]
Most changes are good, I just prefer UK spellings over US as somebody from England.

**Comment by FelixFromDiscord** [comment]
man ignoring the existence of english dialects 💀

**Comment by intergrav** [comment]
💀 yup I can look at that once I’m home lmao

File: README.md [hunk 5]
```diff
@@ -22,31 +22,31 @@ Simple and easy to use Minecraft client for 1.8.9 (and in future™, newer versi
...
-This project uses [GNU General Public License v3.0](LICENSE), a copyleft license. This may need to be changed in future due to it technically conflicting with Minecraft.
+## License
+This project uses [GNU General Public License v3.0](LICENSE), a copyleft license. This may need to be changed in the future due to it technically conflicting with Minecraft.
 
-While not stated in the licence, I would appreciate forks having a different name, logo and design. It's fine for feature branches and custom builds though. I'd just rather all of these things changed, not one of them.
+While not stated in the license, I would appreciate forks having a different name, logo, and design. It's fine for feature branches and custom builds though. I'd just rather all of these things changed, not one of them.
 
-There are some parts of the code that use other licences (such as code "borrowed" from other projects). If this is the case, it will be noted in a comment or another licence file in the same directory.
+Some parts of the code use other licenses (such as code "borrowed" from other projects). If this is the case, it will be noted in a comment or another license file in the same directory.
```
[the remark this answers is not in the document]
Comment (intergrav) [reply README.md:33]: In this case, the comma [is necessary](https://www.grammarly.com/blog/comma-before-and/).

**Comment by intergrav** [comment]
Alright, I reverted words to use UK spelling and changed "intended" to "intend" :)

**Review by TheKodeToad — APPROVED** [review]

**Pull request merged**